tests/test_authenticator.py

```diff
@@ -1068,11 +1069,25 @@ def test_create_bearer_token(self):
             bearer_token_signing_secret='foo'
         )
 
-        # A token is created and signed with the bearer token.
         token1 = authenticator.create_bearer_token(oauth1.NAME, "some token")
-        eq_("eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJvYXV0aDEiLCJ0b2tlbiI6InNvbWUgdG9rZW4ifQ.Ve-bbEN4mdWQdR-VA6gbrK2xOz2KRbmPhttmTTCA0ng",
-            token1
-        )
+
+        # We know the bearer token is a JWT-like string that looks
+        # like this. There's no timestamp in the payload, so this won't
+        # change over time.
+        expect_jwtlike = u'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0b2tlbiI6InNvbWUgdG9rZW4iLCJpc3MiOiJvYXV0aDEifQ.tolCFBIKqtp2NORHtwy5-44GwAwDIma2pebPGXvPY-g'
+        eq_(token1, expect_jwtlike)
+
+        # We can't call jwt.decode to peek into expect_jwt, because
+        # it's not a proper JWT and the signature verification will
+        # fail.  But if you're debugging this test and you need to
+        # look at the pieces that make up expect_jwt, you can call
+        # PyJWT._load(), like so:
+        from jwt.api_jwt import _jwt_global_obj
+        loaded = _jwt_global_obj._load(expect_jwtlike)
+        payload = loaded[0]
+        payload_dict = json.loads(payload)
+        eq_("some token", payload_dict['token'])
+        eq_(oauth1.NAME, payload_dict['iss'])
```

Comment:
Similar to the adobe_vendor_id JWT case. The string we generate changes because it's an encoded version of the JSON representation of a dictionary, and an OrderedDict gives a different representation than a dict. I added look-inside code to the test to make it easier to debug in the future.